Tasks/Churo/Task_6/shop_data.py

```diff
@@ -0,0 +1,62 @@
+import decimal
+repo = {100: ["bread", 2], 101: ["bakery products", 3], 102: ["cookie", 5], #{category: [product, price]}
+        200: ["pork", 9.99], 201: ["beaf", 15], 202: ["chicken", 7],
+        300: ["hake fish", 15], 301: ["cod", 25], 302: ["shrimps", 35]}
+
+shoping_cart_final = {}
+total_price = 0
+
+def get_bakery():
+    bakery_repo = {}
+    bakery_repo.update([v for k,v in repo.items() if k in range(100,199,1)])
+    return bakery_repo.items()       
+
+def get_meat():
+    meat_repo = {}
+    meat_repo.update([v for k,v in repo.items() if k in range(200,299,1)])
+    return meat_repo.items()
+
+def get_fish():
+    fish_repo = {}
+    fish_repo.update([v for k,v in repo.items() if k in range(300,399,1)])
+    return fish_repo.items()   
```

Comment:
same goes here, the logic is similar, so it can be implemented as a one method with additional parameter